hardware_tests/test_polarization_controller_thorlabs_mpc320.py

```diff
@@ -75,6 +75,25 @@ def test_move_absolute(device: PolarizationControllerThorlabsMPC320) -> None:
     # device.move_absolute(ChanIdent.CHANNEL_3, 0 * ureg.degree)
 
 
+def test_jog(device: PolarizationControllerThorlabsMPC320) -> None:
+
+    jog_step_1 = 50
+    jog_count_1 = 5
+
+    device.set_params(jog_step_1=jog_step_1, jog_step_2=25, jog_step_3=25)
+
+    device.home(ChanIdent.CHANNEL_1)
+    device.home(ChanIdent.CHANNEL_2)
+    device.home(ChanIdent.CHANNEL_3)
+
+    # Compare jog distance versus absolute distance
```

Comment:
It's best to spell out exactly what is going on here and what we expect. Something like "Validate that we jogged to the expected position. If we are at the correct position, the move_absolute command should not cause the device to move at all." just above line 92.